docs/installation/installation.md

```diff
@@ -1,9 +1,10 @@
 # Installation
 
-* [Using conda](#using-conda)
-* [Using pip from PyPi](#using-pip-from-pypi)
-* [Install from source](#install-from-source)
-* [Testing if it worked](#testing-if-it-worked)
+- [Installation](#installation)
```

Comment:
Is this extra level of indentation necessary for some RTD functionality?

Reply:
Thank you @huddlej and @jameshadfield for the reviews. I did not test the installation instructions when I added them and I assumed it was an issue to be addressed later on.

> Would you like to take a shot at this reordering of instructions and also confirm that they work for you?

Sure! I'll change the instructions and I will test them.

Reply:
> Is this extra level of indentation necessary for some RTD functionality?

No, I think it's a mistake on my end. The current version works fine with the previous indentation.

Reply:
This issue is resolved with the latest changes.